AndroidSDKCore/src/main/java/com/leanplum/internal/JsonConverter.java

```diff
@@ -45,7 +45,7 @@ public static String toJson(Map<String, ?> map) {
     }
     try {
       return mapToJsonObject(map).toString();
-    } catch (JSONException e) {
+    } catch (Throwable e) {
```

Comment:
Is it safe to be returning null everywhere? Could we incur data loss or other issues? Would we end up with NPEs instead of OOMs?

Reply:
This method returned null before. Removed Throwable - added only OutOfMemory catch.